front/src/modules/command-menu/components/CommandMenuItem.tsx

```diff
@@ -44,11 +53,25 @@ export const CommandMenuItem = ({
   };
 
   return (
-    <MenuItemCommand
-      LeftIcon={Icon}
-      text={label}
-      command={shortcuts ? shortcuts.join(' then ') : ''}
-      onClick={onItemClick}
-    />
+    <StyledMenuItem onSelect={onItemClick}>
```

Comment:
mmh ok, so we need to use the cmdk components. Let's actually refactor the MenuItemCommand itself to use cmdk components. (let's move the styles needed within this MenuItemCommand)